.jenkins/pytorch/test.sh

```diff
@@ -26,6 +26,12 @@ if [[ "$BUILD_ENVIRONMENT" == *cuda11* ]]; then
   export BUILD_SPLIT_CUDA=ON
 fi
 
+if [[ "$BUILD_ENVIRONMENT" == *pytorch-linux-bionic-py3.6-clang9* ]]; then
```

Comment:
An advantage to using the potential solution I mentioned in the other comment is we'd be able to have something more clear here, like: 
```suggestion
if [[ "$BUILD_ENVIRONMENT" == *noarch* ]]; then
```